src/test/java/com/slack/kaldb/chunk/ChunkCleanerTaskTest.java

```diff
@@ -29,15 +30,15 @@
   private ChunkManagerUtil<LogMessage> chunkManagerUtil;
 
   @Before
-  public void setUp() throws IOException {
+  public void setUp() throws IOException, TimeoutException {
```

Comment:
Do we want to change the ChunkCleanerTask to use the service manager also?

Reply:
I actually am already working on this in a followup PR here - https://github.com/slackhq/kaldb/pull/64. I figured I would make all the assorted fixes we discussed as part of a standalone PR.